### PR TITLE
Products Page

### PR DESCRIPTION
**Pull request by bajajcodes**
- Products listing page where each card holds image of the product, title, price, description and action buttons.
- Each card is 380 pixels wide which means on single column layout there will be multiple rows and single column of cards and rest will be multiples of rows and columns. 
- Grid using `reaeat` `auto-fill` `minmax` takes care of cards layout.
-  Filters are placed on left of view port on multi-column layout and translated to -1000 X on single-column layout.
- Reset is repeated throughout.

https://user-images.githubusercontent.com/29247011/154841326-c5c2584e-c245-4d95-91c1-912d33a61808.mp4

### CODE REVIEW

File: css/auth.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+:root {
+    --grey-color: #f4f4f4;
+    --dark-grey-color: #c4c2c2;
+}
+
+body{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body{
```
Comment (junaidshaikh-js) [inline css/auth.css]: `body` element targeted twice

File: css/products.css [hunk 2]
```diff
@@ -0,0 +1,200 @@
+/* single-column layout */
+body {
```
Comment (junaidshaikh-js) [inline css/products.css]: it is better to use wrapper rather than body element to style layout

File: css/products.css [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    background-color: var(--aside-beautify-color);
+    min-height: 100vh;
+}
+
+.aside-hamburger {
+    background-color: transparent;
+    border: 0;
+    font-size: var(--size-10);
+    cursor: pointer;
+    /* transition: transform 560ms; */
```
Comment (junaidshaikh-js) [inline css/products.css]: code comment should be removed

**Review by junaidshaikh-js — COMMENTED** [review]
Good work!

File: css/auth.css (hunk 1 above)
Comment (bajajcodes) [reply css/auth.css]: I have made the change.

File: css/products.css (hunk 3 above)
Comment (bajajcodes) [reply css/products.css]: Removed :)

File: css/products.css (hunk 2 above)
Comment (bajajcodes) [reply css/products.css]: Wrapper added to body element :+1:

Comment (bajajcodes) [reply css/products.css]: Wrapper added to body element :+1:

**Review by junaidshaikh-js — APPROVED** [review]
Nice, you updated the code with required changes.

File: pages/login.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                                </div>
+
+                    </main>
+
+                    <aside class="aside">
+                            <img src="../assets/images/login-cuate_black.png" alt="">
+                    </aside>
+
+</body>
+</html>
```
Comment (arpit01923) [inline pages/login.html]: Add line here

File: pages/signup.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                                </div>
+
+                    </main>
+
+                    <aside class="aside">
+                            <img src="../assets/images/Sign up-cuate_black.png" alt="">
+                    </aside>
+
+</body>
+</html>
```
Comment (arpit01923) [inline pages/signup.html]: add line here too

File: scripts/aside_navbar.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+asideHamburger.addEventListener("click", () => {
+    if(asideHamburger.firstElementChild.innerText === 'expand_more'){
+        asideHamburger.firstElementChild.innerText = 'expand_less';
+    }else{
+        asideHamburger.firstElementChild.innerText = 'expand_more';
+    }
+    aside.classList.toggle("beautify-aside");
+    asideNavbar.classList.toggle("show");
+});
```
Comment (arpit01923) [inline scripts/aside_navbar.js]: same

**Review by arpit01923 — APPROVED** [review]
Adding line to last is a good practise. rest all looks good

File: scripts/aside_navbar.js (hunk 6 above)
Comment (bajajcodes) [reply scripts/aside_navbar.js]: Done

File: pages/signup.html (hunk 5 above)
Comment (bajajcodes) [reply pages/signup.html]: Done

File: pages/login.html (hunk 4 above)
Comment (bajajcodes) [reply pages/login.html]: Done

**Pull request merged**